kubernetes/base/deployment.yml

```diff
@@ -28,19 +28,24 @@ spec:
             periodSeconds: 5
           readinessProbe:
             httpGet:
-              port: http
+              name: http
+              port: 80
               path: /status/ready
             initialDelaySeconds: 5
             periodSeconds: 10
             timeoutSeconds: 2
+          ports:
+            - containerPort: 80
           envFrom:
           - configMapRef:
               name: <% .Name %>-config
           env:
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
+          - name: SERVER_PORT
+            value: "80"
```

Comment:
just curious why do we put backend on 80?  would it make it more confusing what is being mapped to what? (eg hides the ingress mapping and stuff)

Reply:
Because it's serving HTTP traffic, and that is the default port. Since you don't need to deal with ports conflicting in Kubernetes this isn't a problem.
What do you mean about hiding the ingress mapping?

Reply:
as in `boilerplate backend port: 80` may lead people to think the `port in the deployment` is the `port for the end url` (which in this case it is)
Kinda hides the fact that ingress will expose it at 80 regardless

Reply:
If someone fundamentally doesn't understand the difference between deployment, service, and ingress, we should try to solve that with documentation, not hoping that they get confused by all the different ports and look it up.

Reply:
Also we are giving them a nice name to refer to, though I realize I put it in the wrong spot above. Will fix that.